GitOut/Features/Git/Log/GitLogViewModel.cs

```diff
@@ -451,6 +467,35 @@ await foreach (GitHistoryEvent stashEntry in Repository.StashListAsync())
             }
         }
 
+        private async Task SquashCommitAsync(LogEntriesViewModel? gte)
+        {
+            IsWorking = true;
+            string subject = gte!.Root.Event.Subject;
+            string body = gte.Root.Event.Body;
+            var branch = GitBranchName.CreateLocal($"gitout-bkp/{Guid.NewGuid():N}");
+            await Repository.CreateBranchAsync(branch);
+            await Repository.ResetToCommitAsync(gte!.Root.Event.Id);
+            await Repository.AddAllAsync();
+            await Repository.CommitAsync(GitCommitOptions.AmendLatest(body.Length > 0 ? $"{subject}{Environment.NewLine}{Environment.NewLine}{body}" : subject));
+            snack.ShowSuccess("Successfully reset to previous commit");
+            await CheckRepositoryStatusAsync();
+            IsWorking = false;
+            const string approveActionText = "YES";
```

Comment:
we could show a 'NO' option as well, even though the `x` does it.

Reply:
That's why I didn't want the NO, because it's covered by the X ^^
I can add it, it might be better to have multiple ways of doing this